src/main/java/seedu/resireg/MainApp.java

```diff
@@ -76,10 +76,13 @@ public void init() throws Exception {
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         Optional<ReadOnlyResiReg> resiRegOptional;
         ReadOnlyResiReg initialData;
+        boolean allowRoomCreation = true;
```

Comment:
Also a nit on my part, should this be `isRoomCreationAllowed` following CS2103's conventions?

Reply:
Alright changed it, thanks for the catch